### PR TITLE
Changes event earliest_start to use time helpers

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Changed Santa event earliest start from 33 minutes & 20 seconds to 30 minutes. Changed shuttle loan earliest start from 6 minutes & 40 seconds to 7 minutes.
/:cl:

@vuonojenmustaturska showed me that you can use time helpers instead of deciseconds - this should make event start times readable more easily.

That aside, some events had earliest_start defined as the default 20 minutes. I removed it from the following: heart_attack, revenant, wizard/blobies.

Tweaks:
- Changed Santa event start from 33 minutes & 20 seconds (20.000 deciseconds) to 30 minutes
- Changed shuttle loan event shuttle loan event start from 6 minutes & 40 seconds (4.000 deciseconds) to 7 minutes.

### CODE REVIEW

**Comment by optimumtact** [comment]
:neutral_face:

File: code/modules/events/wizard/ghost.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	weight = 2
 	typepath = /datum/round_event/wizard/possession
 	max_occurrences = 5
-	earliest_start = 0
+	earliest_start = 0 MINUTES
```
Comment (ShizCalev) [inline code/modules/events/wizard/ghost.dm]: why

File: code/modules/events/wizard/blobies.dm [hunk 2]
```diff
@@ -3,7 +3,6 @@
 	weight = 3
 	typepath = /datum/round_event/wizard/blobies
 	max_occurrences = 3
-	earliest_start = 12000 // 20 minutes (gotta get some bodies made!)
```
Comment (ShizCalev) [inline code/modules/events/wizard/blobies.dm]: changing behavior.

File: code/modules/events/heart_attack.dm [hunk 3]
```diff
@@ -3,7 +3,6 @@
 	typepath = /datum/round_event/heart_attack
 	weight = 20
 	max_occurrences = 2
-	earliest_start = 12000
```
Comment (ShizCalev) [inline code/modules/events/heart_attack.dm]: changing behavior.

File: code/modules/antagonists/revenant/revenant_spawn_event.dm [hunk 4]
```diff
@@ -5,7 +5,6 @@
 	typepath = /datum/round_event/ghost_role/revenant
 	weight = 7
 	max_occurrences = 1
-	earliest_start = 12000 //Meant to mix things up early-game.
```
Comment (ShizCalev) [inline code/modules/antagonists/revenant/revenant_spawn_event.dm]: changing behavior.

**Comment by ShizCalev** [comment]
Corrected the times in the OP (since dealing with fractions of a minute is silly, just put the actual time instead), I hope you don't mind.

**Comment by 81Denton** [comment]
That's fine

File: code/modules/events/wizard/ghost.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/events/wizard/ghost.dm]: In case someone makes a future PR that increases the time to >0

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
One day we'll un-msoify these defines @optimumtact 

One day...

**Pull request merged**